### PR TITLE
Utm source

### PR DESCRIPTION
**Pull request by ramonsnir**

### CODE REVIEW

File: themes/jamstackthemes/layouts/index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h1>New Themes <span class="count">(<span class="count-number">{{ $themeCount }}
   </div>
 
   <div>
-    <div class="blurb blurb-desktop"><p>A list of themes and starters for <a href="https://jamstack.org/" target="_blank" rel="noopener">JAMstack</a> sites. We're the largest gallery of user submitted, open source themes for static site generators 🦖. This site is maintained with <a href="https://github.com/stackbithq/jamstackthemes#contribution-guidelines">community contributions</a> and with the kind support of <a href="https://www.stackbit.com">stackbit.com.</a></p></div>
+    <div class="blurb blurb-desktop"><p>A list of themes and starters for <a href="https://jamstack.org/" target="_blank" rel="noopener">JAMstack</a> sites. We're the largest gallery of user submitted, open source themes for static site generators 🦖. This site is maintained with <a href="https://github.com/stackbithq/jamstackthemes#contribution-guidelines">community contributions</a> and with the kind support of <a href="https://www.stackbit.com/&utm_source=jamstackthemes&utm_medium=referral&utm_campaign=blurb">stackbit.com.</a></p></div>
```
Comment (JugglerX) [inline themes/jamstackthemes/layouts/index.html:33]: `utm_campaign` should be `jamstackthemes`
`utm_content` should be `blurb`

also first param should be a `?` ie ?utm_source=

File: themes/jamstackthemes/layouts/partials/stackbit.html [hunk 2]
```diff
@@ -4,4 +4,4 @@
   <img src="{{ "images/stackbit.png" | relURL }}" alt="Stackbit Logo" />
 </div>
 
-<span class="made-by made-by-mobile"><a href="https://www.stackbit.com"><img src="{{ "images/stackbit.png" | relURL }}" alt="Stackbit Logo" /></a></span>
\ No newline at end of file
+<span class="made-by made-by-mobile"><a href="https://www.stackbit.com/&utm_source=jamstackthemes&utm_medium=referral&utm_campaign=footer"><img src="{{ "images/stackbit.png" | relURL }}" alt="Stackbit Logo" /></a></span>
```
Comment (JugglerX) [inline themes/jamstackthemes/layouts/partials/stackbit.html:7]: same as above. `utm_campaign` should be `jamstackthemes` and `utm_content` should be `footer`

**Review by JugglerX — CHANGES_REQUESTED** [review]

File: themes/jamstackthemes/layouts/index.html (hunk 1 above)
Comment (ramonsnir) [reply themes/jamstackthemes/layouts/index.html:33]: you want both source and campaign to have the same value?

Comment (JugglerX) [reply themes/jamstackthemes/layouts/index.html:33]: yes, unless you have a better suggestion for the campaign value.

Comment (JugglerX) [reply themes/jamstackthemes/layouts/index.html:33]: we could also just remove the campaign value. its not required. and it feels like we are trying to make it fit here when its not needed.

**Review by JugglerX — APPROVED** [review]

**Pull request merged**